src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas2/Cas2ReportsTest.kt

```diff
@@ -155,14 +209,16 @@ class Cas2ReportsTest : IntegrationTestBase() {
         withId(event1Id)
         withType(DomainEventType.CAS2_APPLICATION_SUBMITTED)
         withData(objectMapper.writeValueAsString(event1ToSave))
-        withOccurredAt(old.atOffset(ZoneOffset.ofHoursMinutes(0, 0)))
+        withOccurredAt(oldSubmitted.atOffset(ZoneOffset.ofHoursMinutes(0, 0)))
```

Comment:
Please confirm the event `occurred_at` field should be populated with submission timestamps?  `domain_events.occurred_at` is what the SQL query is using to filter the reports for the last 12 months (see comment above).

Reply:
Resolved above https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/1516#discussion_r1498937736